src/components/directory-content.js

```diff
@@ -744,6 +837,22 @@ const DirectoryContent = () => {
                                             setOpenScriptContingencyDialog(
                                                 true
                                             );
+                                        } else if (
```

Comment:
Maybe we should add a `handleRowClick` function for readiness which take a `event.rowData` as parameter.